pallets/utxo/rpc/src/lib.rs

```diff
@@ -93,4 +101,22 @@ where
             data: Some(format!("{:?}", e).into()),
         })
     }
+
+    fn nft_read(
+        &self,
+        at: Option<<Block as BlockT>::Hash>,
+        id: H256,
+    ) -> Result<Option<(/* Data url */ Vec<u8>, /* Data hash */ [u8; 32])>> {
+        let api = self.client.runtime_api();
+        let at = BlockId::hash(at.unwrap_or_else(||
+            // If the block hash is not supplied assume the best block.
+            self.client.info().best_hash));
+
+        let runtime_api_result = api.nft_read(&at, id);
```

Comment:
Could you please explain what's going on after this line?

Reply:
It's just a standard mechanism to call the API, actually, it will call `pallets/utxo/src/lib.rs:1083`:

```rust
    pub fn nft_read(
        id: H256,
    ) -> Option<(
        /* Data url */ frame_support::inherent::Vec<u8>,
        /* Data hash */ [u8; 32],
    )> {
        match Nft::<T>::get(id)? {
            TokenInstance::Nft {
                data_hash,
                data_url,
                ..
            } => Some((data_url, data_hash)),
            _ => None,
        }
    }
}
```